src/varnish.c

```diff
@@ -152,7 +152,17 @@ static int varnish_monitor(void *priv,
 
   conf = priv;
 
-#if HAVE_VARNISH_V4
+#if HAVE_VARNISH_V5
+  char namebuff[100];
+  char *c;
+
+  c = rindex(pt->name, '.');
+  strcpy(namebuff, c + 1);
+  name = namebuff;
+
+  (void)class;
```

Comment:
`class` seems to only be *read* in the `HAVE_VARNISH_V3` block. Can we remove this from all the other implementations, including this one to simplify the code?